README.md

```diff
@@ -45,23 +47,31 @@ cd stac-fastapi
 pip install -e \
     stac_fastapi/api \
     stac_fastapi/types \
-    stac_fastapi/extensions \
-    stac_fastapi/sqlalchemy
+    stac_fastapi/extensions
+
+# Install a backend
+pip install -e stac_fastapi/sqlalchemy
+# or
+pip install -e stac_fastapi/pgstac
 ```
 
 ## Local Development
 Use docker-compose to deploy the application, migrate the database, and ingest some example data:
 ```bash
 docker-compose build
 docker-compose up
+
+# You can also launch specific application
+docker-compose up app-sqlalchemy  # or sqlalchemy-loadjoplin
```

Comment:
Maybe describe where these names come from? That you can look them up in `docker-compose.yml`? Also, seems weird to have `app-{sqlalchemy,pgstac}` but `{sqlalchemy,pgstac}-loadjoplin`. Maybe switch names to `loadjoplin-{sqlalchemy,pgstac}`?